source/installation/resource-manager/sge.rst

```diff
@@ -93,3 +93,20 @@ This sets the sh shell to behave like bash and ensures you've sourced your users
             set +o posix
             . ~/.bashrc
             %s
+
+Invalid Job name
+****************
+You're likely to encounter an issue in running batch connect applications complaining about invalid
+job names like the error below.
+
+``Unable to read script file because of error: ERROR! argument to -N option must not contain /``
+
+To resolve this set ``OOD_JOB_NAME_ILLEGAL_CHARS`` to ``/`` for all OOD applications in the 
+``/etc/ood/profile`` file.
```

Comment:
`/etc/ood/profile` is not the correct place for these type of modifications. Rather, `/etc/ood/config/nginx_stage.yml` and `pun_custom_env`

We might need to add clarification to the documentation in https://osc.github.io/ood-documentation/master/customization_overview.html to move `/etc/ood/profile` to clarify the first items are for changing the environment of the per user NGINX for various OnDemand configuration options, and that /etc/ood/profile is special for changing things like SCLs to load, etc. I can do that.